feature/experimental/bgp/otg_tests/bgp_tcp_mss_path_mtu/README.md

```diff
@@ -0,0 +1,38 @@
+# RT-1.21: BGP TCP MSS and PMTUD 
```

Comment:
Please use the following instead

# RT-1.21: BGP TCP MSS and PMTUD 

## Summary

- Validate changes in TCP MSS value is allowed and takes effect
- Validate DUT's PMTUD compliance

## Topology

DUT1:port1 <-> port1:ATE:Port2 <-> port1:DUT2

## Procedure
- Establish BGP sessions as follows:
	- ATE:port2 --- eBGP-IPv4/IPv6 ---- DUT2:port1
	- DUT1:port1 ---- iBGP IPv4 ---- DUT2:port1.
- TODO : Verify that the default TCP MSS value is set below the default interface MTU value.
- Change the Interface MTU on the DUT2:port1 port as well as ATE:port2 to 5040B
- Configure IP TCP MSS value of 4096 bytes on the DUT2:port1.
- Re-establish the EBGP sessions by tcp reset.
- Verify that the TCP MSS value is set to 4096 bytes for the IPv4 and IPv6 EBGP sessions.
- Establish iBGP session with MD5 enabled between DUT1:port1 and DUT2:port2.
- Ensure that the MTU on the ATE:port1 towards DUT1:port1 is set at defualt (1500B) while the DUT1:port1 interface towards ATE:port1 is set at 5040B.
- Enable PMTUD on DUT2.
- Re-establish the IBGP sessions by tcp reset
	- TODO : Validate that the min MSS value has been adjusted to be below 1500 bytes on the tcp session.

Reply:
updated README and also script as per the these teststeps.